cmd/local/local.go

```diff
@@ -65,8 +65,9 @@ func NewCommand() *cobra.Command {
 	// todo: UPDATE IT BEFORE MERGING
 	// todo: UPDATE IT BEFORE MERGING
 	// todo: UPDATE IT BEFORE MERGING
-	localCmd.Flags().StringVar(&gitOpsBranch, "gitops-branch", "add-ingress-localhost", "version/branch used on git clone")
```

Comment:
I guess we should keep this hardcoded reference until we merge that gitops-template branch, wdyt?

Reply:
hey @converge, with the last PR that @6za did, now we just need to specify gitops-branch when calling the local installation if we want to test a different branch, like following:

`./kubefirst local --gitops-branch add-ingress-localhost`

Reply:
not sure if I got it, we can do that with any flag available since the beginning of the command...

no preference, we can use --gitops-branch as well